### PR TITLE
Forms

### PR DESCRIPTION
**Pull request by edubew**
### This project milestone fulfills the following project requirements:
- Create a method called` current_user` in your ApplicationController to make `current user` data available to all controllers.
    - It will return the first user from the database.
- Create forms to perform the following functions:
    - Creates a Post on behalf of the current_user (use the method that you created in your ApplicationController).
    - Create a comment on behalf of the current_user (use the method that you created in your ApplicationController).
- Allow Users to add likes to Posts.

### CODE REVIEW

File: app/views/posts/index.html.erb [hunk 1]
```diff
@@ -27,6 +28,8 @@
             </div>
           <% end %>
         </div>
+        <a href="<%= @user.id%>/posts/" class="button">Add Post</a>
```
Comment (Whoistolu) [inline app/views/posts/index.html.erb]: - [ ] [REQUIRED] Great job with all the implementations @edubew , your implementations work well except for the routing error I get when I click on the `Add post` button in an attempt to create a post in the form.
### To fix this:
- In order to access an individual post, use:
``` ruby
<%= link_to  user_post_path(user_id: @user.id) do %>   
```
Let's also ensure the `Add Post` button is not overlapping anymore in order to have a more professional design.
Screenshot 2023-06-17 at 21 32 25

Screenshot 2023-06-17 at 21 36 17

**Review by Whoistolu — CHANGES_REQUESTED** [review]
Hi @edubew 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

- [ ]  _Check the inline comments under the review for sections to improve._

### Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

### N/A

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/views/posts/index.html.erb (hunk 1 above)
Comment (edubew) [reply app/views/posts/index.html.erb]: Hello @Whoistolu ,
Thank you for your invaluable feedback and insights on correcting the error. The button now works as it should. I really do appreciate the support you have provided

**Review by Lucas-Masaba — APPROVED** [review]
# Hello @edubew 👋 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

almost-there

## Highlights 🍾 
- Linter checks passing :heavy_check_mark: 
- Made changes as requested by the previous reviewer :+1: 

## Optional suggestions 💭 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏 💻 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Lucas-Masaba in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**